src/DispatchFunctional.jl

```diff
@@ -54,7 +54,7 @@ end
 
 libxc_energy(terms, ρ) = haskey(terms, :zk) ? reshape(terms.zk, 1, size(ρ, 2)) .* ρ : false
 
-function DftFunctionals.potential_terms(func::LibxcFunctional{:lda}, ρ::Matrix{Float64})
+function DftFunctionals.potential_terms(func::Union{LibxcFunctional{:lda},LibxcFunctional{:hyb_lda}}, ρ::Matrix{Float64})
```

Comment:
Line too long (same as many others ... we try to limit it to something like 92).